airflow-core/src/airflow/ui/public/i18n/locales/hi/admin.json

```
@@ -0,0 +1,167 @@
+{
+  "columns": {
+    "description": "विवरण",
+    "key": "कुंजी",
+    "name": "नाम",
+    "value": "मान"
+  },
+  "config": {
+    "columns": {
+      "section": "सेक्शन"
+    },
+    "title": "Airflow विन्यास"
+  },
+  "connections": {
+    "add": "कनेक्शन जोड़ें",
+    "columns": {
+      "connectionId": "कनेक्शन ID",
+      "connectionType": "कनेक्शन प्रकार",
+      "host": "होस्ट",
+      "port": "पोर्ट"
+    },
+    "connection_one": "कनेक्शन",
+    "connection_other": "कनेक्शन्स",
+    "delete": {
+      "deleteConnection_one": "1 कनेक्शन हटाएं",
+      "deleteConnection_other": "{{count}} कनेक्शन्स हटाएं",
+      "firstConfirmMessage_one": "आप निम्नलिखित कनेक्शन को हटाने वाले हैं:",
+      "firstConfirmMessage_other": "आप निम्नलिखित कनेक्शन्स को हटाने वाले हैं:",
+      "title": "कनेक्शन हटाएं"
+    },
+    "edit": "कनेक्शन संपादित करें",
+    "form": {
+      "connectionIdRequired": "कनेक्शन ID आवश्यक है",
+      "connectionIdRequirement": "कनेक्शन ID में केवल स्थान नहीं हो सकते",
+      "connectionTypeRequired": "कनेक्शन प्रकार आवश्यक है",
+      "extraFields": "अतिरिक्त फ़ील्ड",
+      "extraFieldsJson": "अतिरिक्त फ़ील्ड JSON",
+      "helperText": "कनेक्शन प्रकार गायब है? सुनिश्चित करें कि आपने संबंधित Airflow Providers Package इंस्टॉल किया है।",
+      "helperTextForRedactedFields": "संपादित फ़ील्ड ('***') अपरिवर्तित रहेंगे यदि संशोधित नहीं किए गए।",
+      "selectConnectionType": "कनेक्शन प्रकार चुनें",
+      "standardFields": "मानक फ़ील्ड"
+    },
+    "nothingFound": {
+      "description": "पर्यावरण वेरिएबल या सीक्रेट मैनेजर के माध्यम से परिभाषित कनेक्शन यहां सूचीबद्ध नहीं हैं।",
+      "documentationLink": "Airflow दस्तावेज में और जानें।",
+      "learnMore": "ये रनटाइम पर हल किए जाते हैं और UI में दृश्यमान नहीं हैं।",
```

```suggestion
      "learnMore": "ये तभी लोड किए जाते हैं जब कोई टास्क्स, जो इन्हें उपयोग करता है, चलाया जाता है और इसलिए ये इस दृश्य में दिखाई नहीं देते।",
```
probably? No strong suggestion, but the word `रनटाइम ` led me to think to clear this out